packages/itwin/tree-widget/src/components/trees/repositories-tree/UseRepositoriesHierarchyProvider.ts

```diff
@@ -51,10 +53,12 @@ export function UseRepositoriesHierarchyProvider({ accessToken, itwinId, setIsLo
             }
           }
         },
+        setHierarchyFilter() {
+          hierarchyChanged.raiseEvent();
```

Comment:
Do you see any issues without this?

Reply:
This was the only problem after updating both hierarchy dependancies.

Reply:
Please file an issue and add a comment here pointing to that issue.

Reply:
Added an issue.